packages/vaex-core/vaex/column.py

```diff
@@ -107,6 +113,14 @@ def __len__(self):
     def trim(self, i1, i2):
         return ColumnIndexed(self.df, self.indices[i1:i2], self.name)
 
+    def __arrow_array__(self, type=None):
+        # TODO: without a copy we get a buserror
+        values = self[:]
+        if hasattr(values, "to_arrow"):
+            return values.to_arrow()
+        else:
+            return pa.array(values)
```

Comment:
The bus error is probably a stackoverflow as `pa.array` would otherwise call the same `__arrow__array__()` again

Reply:
Yes, I think so too. I tried to reproduce how I got it, but failed :(
Thanks for the review!